components/Navigation/index.js

```diff
@@ -12,7 +12,7 @@ export default function Navigation() {
           create
         </StyledCenterButton>
         <StyledSideButton onClick={() => (window.location.href = "/calendar")}>
```

Comment:
it's better to use the `useRouter` hook from Next.js here, since this hook is optimized for Next.js Apps